participants/tanimal.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Tilman Adler",
+  "company": "Method Park",
+  "when": {
+    "friday": true,
+    "saturday": true
+  },
+  "tags": ["typescript", "react", "angular", "svelte", "vanilla"],
+  "vegan": false,
+  "vegetarian": false,
+  "allergies": "",
+  "what_is_my_connection_to_javascript": "Isn't this that thing typescript compiles to?",
+  "what_can_i_contribute": "What I learnt building covid-karte.de with (almost) only vanilla js",
+  "tshirt": "M-M",
+  "twitter": "t.animal"
```

Comment:
```suggestion
  "twitter": "t_animal"
```

I think the red test actually caught an error here (Twitter handle ≠ Twitter display name).

Reply:
This is how the test output looked to me, before I figured out the results were loaded via xhr. Stupid me with my stupid adblocker.

![screenshot](https://user-images.githubusercontent.com/1775745/171627114-10562e80-eb2a-4cdb-8fce-56e8254bf362.png)